ReadMe.md

```diff
@@ -2,15 +2,18 @@
 
 Developed by Brett208 (Vagabond) and Hooman
 
-A console application to review the publicly exported information contained in Outpost 2 mission DLLs. Written using C++17 features with Visual Studio 2017 project files. MissionScanner only supports x86 compilation. For more information on Outpost 2, visit https://outpost2.net.
+A console application to review the publicly exported information contained in Outpost 2 mission DLLs. Written using C++17 features. Tested on Windows and Linux. Windows project files compile using MSVC with Visual Studio 2017. A Linux makefile is also included. MissionScanner fully supports x86 and x64 compilation. For more information on Outpost 2, visit https://outpost2.net.
 
-### Usage
+## Usage
 
-MissionScanner (archivename.(vol|clm) | directory)...
+MissionScanner (archivename.(vol|clm) | directory)... [-L]
 
-## Example Commands
+#### Optional Arguments
```

Comment:
Looks like this should be header level `###` (unless intentional?)

Reply:
This was intentional because I didn't like the amount of spacing created by the higher header level for the small amount of information presented. I'm not overly attached either way if you think it makes better logical sense or looks better with the ### header, I can switch it.

Reply:
That's fine. Let's just leave it then. The only other option I can see, is to insert an extra level of heading in there somewhere, though that may be kind of artificial.